src/tests/conftest.py

```diff
@@ -67,6 +67,18 @@ def ip():
     runsql(ip_session, "DROP TABLE author")
 
 
+@pytest.fixture
+def ip_wo_engine():
+    """Provides an IPython session in which tables have been created"""
+    ip_session = InteractiveShell()
+    ip_session.register_magics(SqlMagic)
+    ip_session.register_magics(RenderMagic)
+    ip_session.register_magics(SqlPlotMagic)
+    ip_session.register_magics(SqlCmdMagic)
+
+    yield ip_session
```

Comment:
please rebase, cause I encountered the same problem in another PR and added a fixture that looks the same, it's called `ip_empty`

Reply:
Gotcha!
[192dd14](https://github.com/ploomber/jupysql/pull/117/commits/192dd14d68deea5d05b69656a33647a229c577b5)